mteb/benchmarks/benchmarks.py

```diff
@@ -438,6 +438,55 @@
     contacts=["gowitheflow-1998"],
 )
 
+MTEB_ARA = Benchmark(
+    name="MTEB(ara, v1)",
+    tasks=MTEBTasks(
+        get_tasks(
+            languages=["ara"],
+            tasks=[
+                # Classification
+                "AJGT",
+                "HotelReviewSentimentClassification",
+                "LanguageClassification",
+                "MassiveIntentClassification",
+                "MultiHateClassification",
+                "MultilingualSentimentClassification",
+                "OnlineStoreReviewSentimentClassification",
+                "RestaurantReviewSentimentClassification",
+                "TweetEmotionClassification",
+                "TweetSarcasmClassification",
+                "TweetSentimentClassification",
```

Comment:
There are a lot of sentiment/hate-speech datasets here - I would probably reduce it down to a few high-quality cases